api/server/router/volume/volume.go

```diff
@@ -44,6 +44,14 @@ func (r *router) initRoutes() {
 			handlers.NewSchemaValidator(nil, schema.ServiceVolumeMapSchema, nil),
 		),
 
+		// get all volumes from all services with attachments
+		httputils.NewGetRoute(
```

Comment:
Cool. Yeah, the `volumes` method already checks the query string:

![image](https://cloud.githubusercontent.com/assets/101085/14829296/368906dc-0bb0-11e6-90a8-3fb6bac79503.png)